### DISCUSSION TITLE
Is Ventoy support Windows 11?

### DISCUSSION THREAD

**Discussion by ghost** [ventoy/Ventoy#1166] [Q&A]
Is Ventoy support Windows 11?

**Comment by ventoy** [comment]
Yes. 
Also you can bypass the hardware check in Windows 11 when install.
https://www.ghacks.net/2021/10/19/ventoy-1-0-55-bypass-windows-11-requirements-check-during-installation/

**Reply by ventoy** [reply]
Or do you mean Ventoy can run in Windows 11 or Ventoy can boot and install Windows 11 ?

**Reply by ghost** [reply]
Ventoy can run on Windows 11?

**Reply by ventoy** [reply]
yes

**Reply by ghost** [reply]
I can install ventoy in windows 11 because my computer crashed when I was installing it to Windows 10

**Reply by kornm123** [reply]
I checked and rechecked:
I try to create a Ventoy drive in Windows 11 and it does not work at all.
The same process works 100% on W10.
I tried also to clear Ventoy from an USB. No luck!!!!
Very curious.

Marcel

**Comment by britainlondon** [comment]
i think ventoy not support win11,  when i use it , it report:
No bootfile found for UEFI

i do not know why?


can you help me?

**Reply by kornm123** [reply]
I do not thik your question is clear enough. 
Did you download Ventoy and ran it by clicking on the exe file?
Did it create a bootable USB drive under W11?
